examples/with-realm-web/pages/index.js

```diff
@@ -0,0 +1,91 @@
+import useSWR from 'swr'
+import { generateAuthHeader, REALM_GRAPHQL_ENDPOINT } from '../lib/RealmClient'
+
+const FIND_MOVIES = `
+  query FindMovies{
+    movies(query: { year: 2014, rated: "PG" } ) {
+      title
+      year
+      runtime
+    }
+  }
+`
+
+const fetcher = async (query) =>
+  fetch(REALM_GRAPHQL_ENDPOINT, {
+    method: 'POST',
+    headers: await generateAuthHeader(),
+    body: JSON.stringify({ query }),
+  }).then((res) => res.json())
+
+const IndexPage = () => {
+  const { data } = useSWR(FIND_MOVIES, fetcher)
+
+  if (data && data.error) {
+    console.error(data.error)
+    return <p>An error occurred: ${data.error}</p>
+  }
+  const movies = data ? data.data.movies : null
+
+  return (
+    <>
+      <div className="App">
+        <h1>"PG" Rated Movies - 2014</h1>
+
+        {data && !movies && <div className="status">No movies Found</div>}
```

Comment:
```suggestion
	        {data && !movies && <div className="status">No movies Found</div>}
```

Reply:
I was checking for `data && !movies` to ensure it has finished fetching data and actually didn't find any movies, 
Updated the check to show fetching data while it's loading ( data is undefined till swr is finished loading, saw `loading = !data` in one of the examples )
```
        {data ? (
          !movies && <div className="status">No movies Found</div>
        ) : (
          <div className="status"> Fetching data...</div>
        )}
```

Is this alright?

Reply:
Yeah that's better than before!